### DISCUSSION TITLE
Comparing Planetary Computer with Google Earth Engine

### DISCUSSION THREAD

**Discussion by gilbertocamara** [microsoft/PlanetaryComputer#195] [General]
Dear all, recently I have been invited to present my views on the differences between MPC and GEE. Here are some links to these discussions to those interested in the topic:

* Podcast episode in Robin Cole's Substack on Deep Learning for Satellite Images: 
         https://www.satellite-image-deep-learning.com/p/satellite-image-time-series-with
* Presentation at WILDLABS Variety Hour 
         https://www.dropbox.com/s/mftys3s0qt0e6n9/WildLabs2023_GCamara_final.pptx?dl=0
* Twitter disccussions
         https://twitter.com/gcamara/status/1619078022581325824
         https://twitter.com/gcamara/status/1619787446555938817

Best
Gilberto

**Comment by TomAugspurger** [comment]
Thanks for sharing! I'm partway through the podcast, and it looks like the WILDLABS video is [online](https://www.youtube.com/watch?v=9QfC_sC5oLE) now.
